### PR TITLE
Update netutils image

### PR DESCRIPTION
**Pull request by ilappe**
## Related Content Pull Request
Related PR: https://github.com/demisto/content/pull/29576

## Related Issues
Related: https://jira-hq.paloaltonetworks.local/browse/CIAC-861

## Description
Updated the netutils image to include the following images:
```
netutil
nmap
iputils
websockets
dnspython
```

- **127.1 mb size**

should be replace the current image in the following yml files:
```
Packs/CommonScripts/Scripts/Ping/Ping.yml
Packs/Nmap/Integrations/Nmap/Nmap.yml
Packs/CommonScripts/Scripts/Dig/Dig.yml
Packs/CommonScripts/Scripts/Ping/Ping.yml
Packs/DeveloperTools/Scripts/CompareIndicators/CompareIndicators.yml
Packs/FeedPublicDNS/Integrations/FeedPublicDNS/FeedPublicDNS.yml
Packs/FiltersAndTransformers/Scripts/IPv4Blacklist/IPv4Blacklist.yml
Packs/FiltersAndTransformers/Scripts/IPv4Whitelist/IPv4Whitelist.yml
Packs/FiltersAndTransformers/Scripts/IsNotInCidrRanges/IsNotInCidrRanges.yml
Packs/FiltersAndTransformers/Scripts/IsRFC1918Address/IsRFC1918Address.yml
Packs/Nmap/Integrations/Nmap/Nmap.yml
Packs/Tufin/Integrations/Tufin/Tufin.yml
```

image

### CODE REVIEW

**Comment by ilappe** [comment]
@JudahSchwartz  will try to add it also to the native image as well

File: docker/netutils/Pipfile [hunk 1]
```diff
@@ -8,6 +8,7 @@ verify_ssl = true
 [packages]
 requests = {extras = ["socks"],version = "*"}
```
Comment (JudahSchwartz) [inline docker/netutils/Pipfile:9]: could we switch to poetry?

Comment (JudahSchwartz) [inline docker/netutils/Pipfile:9]: can we add websockets/ntlm/dnspython/aiohttp ? 
Lets make sure we're adding here everything we can

**Review by GuyAfik — APPROVED** [review]
@ilappe looks good, awesome work!

**Comment by ilappe** [comment]
> @ilappe looks good, awesome work! add the image to `depandabot.yml`

`netutils` image already defined in `depandabot.yml`

**Pull request merged**